mobify/progressive-web-scaffold
# Update SDK to 0.11.2

Upgrade Scaffold to use SDK v0.11.2!

## Changes
- Upgrade the SDK version to 0.11.2
- Apply fixes to various templates and components (both markup changes and styling changes) to address breaking changes introduced in the SDK version 0.11.0
    - **Note 1**: updated `_variable.scss` to match what's now used in the SDK; in particular the neutral colors. _This could impact currently open PRs..._
    - **Note 2**: some color utility classes changed across various pages, particularly check! _This may impact currently open PRs..._
- Update the `build-sprites` npm command to reflect the changes to the icon sprite's new name space (no longer `shoppicon`, now `pw`)

## How to test-drive this PR
- Run `rm -rf node_modules` in the `/web` directory
- Run `npm install`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify that the site isn't broken!
- Verify that none of the breaking changes [from 0.11.0](https://github.com/mobify/progressive-web-sdk/blob/develop/CHANGELOG.md) are causing any trouble in the scaffold
